### PR TITLE
Fixes problem of adding tenant to model discriminators

### PR DESCRIPTION
**Pull request by felipe-augusto**
When creating a discriminator and using together with node-mongo-tenant, we should also apply tenant to all model discriminators, otherwise tenant does not works.

### CODE REVIEW

**Comment by felipe-augusto** [comment]
@alrik After this is merged, can you please release a new version?

File: index.js [hunk 1]
```diff
@@ -315,6 +315,15 @@ class MongoTenant {
       Object.defineProperty(MongoTenantModel, staticProperty, descriptor);
     }
 
+    // create tenant models for discriminators if they exist
+    if(BaseModel.discriminators) {
```
Comment (alrik) [inline index.js]: Just minor cosmetics: could you be so kind and add a space after `if`.
Thanks :heart:

**Review by alrik — CHANGES_REQUESTED** [review]

**Review by alrik — APPROVED** [review]

**Comment by alrik** [comment]
Refers to #38

**Comment by alrik** [comment]
@felipe-augusto I finally made it to release version 1.5.0 :tada:

**Comment by felipe-augusto** [comment]
Thanks @alrik. Great work!

**Pull request merged**